near-sdk-core/src/code_generator/impl_item_method_info.rs

```diff
@@ -84,7 +90,7 @@ impl ImplItemMethodInfo {
                 method_invocation = quote! {
                     contract.#ident(#arg_list)
                 };
-                if mutability.is_some() && reference.is_some() {
+                if !is_view && reference.is_some() {
```

Comment:
Why do we need this change?

Reply:
just small refactoring to make the code more self-descriptive